ui/src/components/Select.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+
+import CreatableSelect from 'react-select/creatable'
+import { KeyOption, keyOptions } from 'components/SelectData'
+import { ActionMeta, OnChangeValue } from 'react-select'

```

Comment:
Let's move this import up one line so that 3rd party imports come before our own imports for better readability/searchability. Its an unwritten rule but most repos follow that pattern